lib/src/collections.dart

```diff
@@ -43,8 +43,9 @@ class CollectionChanges {
   final List<int> modifications;
   final List<int> modificationsAfter;
   final List<Move> moves;
+  final bool cleared;
```

Comment:
```suggestion
  final bool isCleared;
```

Reply:
Or should it be `wasCleared` then, as it has already happened. I'm okay with any of those, but probably prefer just `cleared`.

Reply:
I will rename it to `isCleared`, to be known that it is a bool value. Otherwise, `cleared' could be thought that it would return the indexes of the cleared items.

Reply:
Okay, also matches dotnet https://www.mongodb.com/docs/realm-sdks/dotnet/latest/reference/Realms.ChangeSet.html